new/LIT7082QeneEmnahsa.xml

```diff
@@ -0,0 +1,86 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7082QeneEmnahsa" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">እምናህሰ፡ ኢሳይያስ፡ ዘወጽአ፡ በአመተ፡ ፍዳ፡ ሃመል፡ ለንጽ፡ ፈረሰ፡ ዖዝያን፡ ሀመልሚል።</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾƎm-nāhsa ʾIsāyəyās za-waṣʾa ba-ʾamata fəddā hāmal lanṣ farasa ʿOzyān hamalmil (Qǝne of the mawaddəs-type)</title>
+                <title xml:lang="en" corresp="#t1">Aries, linen, the green horse of Uzziah, which on the year of judgment came out from under Isaiah's roof… (?)</title>
+                <editor role="generalEditor" key="AB"/>
```

Comment:
I do not understand it eather, I do not see relations. Literally, it might be, in my opinion: "From the fringe of the robe  Isaiah, which came out in the year of judgement, Ram, bandage, green horse of Uzziah...". This all is bewildering and difficult to penetrate. But you are not obliged to quote the line in full,  you can quote as the tile a part, e.g., just እምናህሰ፡ ኢሳይያስ፡ ዘወጽአ፡ (is this my impression of qenes of Mr Gabra Le'ul are quite difficult?).

Reply:
For me, Gabra Le'ul's qenes are the first ones I really engage with, so I can't compare. But if you say they are difficult, I believe you :)